environment/metadata.go

```diff
@@ -166,6 +172,25 @@ func fillEC2PluginTests(e *MetaData, data *MetaDataStrings) {
 	e.EC2PluginTests = m
 }
 
+func fillExcludedTests(e *MetaData, data *MetaDataStrings) {
+	if e.ComputeType != computetype.EC2 {
+		return
+	}
+
+	if len(data.ExcludedTests) == 0 {
+		log.Println("Testing all EC2 plugins")
+		return
+	}
+
+	plugins := strings.Split(strings.ReplaceAll(data.ExcludedTests, " ", ""), ",")
```

Comment:
is the space replace needed are we expecting 
a,b or a, b

Reply:
We are expecting a,b. I folllowed the same pattern as `fillEC2PluginTests` method and it's probably to prevent human errors.